src/targets/gpu/CMakeLists.txt

```diff
@@ -41,11 +41,20 @@ else()
     set(MIGRAPHX_USE_HIPRTC ON CACHE BOOL "Use hipRTC APIs")
 endif()
 
-include(Embed)
 file(GLOB KERNEL_FILES CONFIGURE_DEPENDS
     ${CMAKE_CURRENT_SOURCE_DIR}/kernels/include/migraphx/kernels/*.hpp)
-message(STATUS "KERNEL_FILES: ${KERNEL_FILES}")
-add_embed_library(migraphx_kernels ${KERNEL_FILES})
+
+if(WIN32)
+    # TODO: re-enable when CK is ported to Windows
+    list(REMOVE_ITEM KERNEL_FILES
+        ${CMAKE_CURRENT_SOURCE_DIR}/kernels/include/migraphx/kernels/ck_gemm.hpp
+        ${CMAKE_CURRENT_SOURCE_DIR}/kernels/include/migraphx/kernels/ck.hpp)
+endif()
+
+if(NOT WIN32)
+    include(Embed)
+    add_embed_library(migraphx_kernels ${KERNEL_FILES})
+endif()
```

Comment:
Windows should use the same `add_embed_library` function, just with a different implementation that uses the resource files instead.

Reply:
I have an idea of how to do this. That will be the separate PR. I will remove `if(NOT WIN32)` from here.